default/scripting/species/common/supply.macros

```diff
@@ -35,4 +41,17 @@ ULTIMATE_SUPPLY
             activation = Planet
             accountinglabel = "ULTIMATE_SUPPLY_LABEL"
             effects = SetMaxSupply value = Value + 3
+            
+[[STANDARD_SUPPLY_GROWTH]]
+'''
+
+STANDARD_SUPPLY_GROWTH
+'''        EffectsGroup // increase 1 per turn, up to max
+            scope = Source
+            activation = And [
+                Planet
+                (Source.LastTurnConquered < CurrentTurn)
+            ]
+            priority = [[AFTER_ALL_TARGET_MAX_METERS_PRIORITY]]
+            effects = SetSupply value = min(Value(Target.MaxSupply), Value + 1)
```

Comment:
No gain for outposts, could move to techs/ship_hulls/SHP_GAL_EXPLO.focs.txt

Reply:
You mean there shouldn't be, or should be an effect for outposts?